tests/tests_build_workflow/opensearch_dashboards/test_build_artifact_opensearch_dashboards_check_plugin.py

```diff
@@ -24,12 +24,13 @@ def __mock(self, props={}, snapshot=True):
                     output_dir="output_dir",
                     name="OpenSearch",
                     version="1.1.0",
+                    patches=["1.0.0"],
```

Comment:
A bit confused here is this saying 1.1.0 can be a patch of 1.0.0?

Reply:
it's just an example for tests, what this says is indeed that 1.1.0 is just 1.0.0 + some changes.

Reply:
Thanks.